test/unit/index.test.js

```diff
@@ -93,12 +93,19 @@ describe('mapWebPacUrlToSCCURL', function() {
   });
 
   it('should return base url if no match is found', function() {
-    const path = 'record=&%!^/';
+    const path = '/record=&%!^/';
```

Comment:
Is `&%!^` just meant to be 'junk'? That wasn't obvious to me... maybe need a comment or something?

Reply:
Added